test/test-unlock.sh

```diff
@@ -19,6 +19,32 @@ begin_test "unlocking a lock by path"
 )
 end_test
 
+begin_test "force unlocking lock with missing file"
+(
+  set -e
+
+  reponame="force-unlock-missing-file"
+  setup_remote_repo_with_file "$reponame" "a.dat"
+
+  GITLFSLOCKSENABLED=1 git lfs lock "a.dat" | tee lock.log
+  id=$(grep -oh "\((.*)\)" lock.log | tr -d "()")
```

Comment:
Definitely outside the scope of this PR, but we should perhaps extract this out to a function, since this `grep | tr` business is a tinge confusing. This one's on me, since I think I wrote this a while ago.